packages/terra-form-select/tests/wdio/select-spec.js

```diff
@@ -1140,6 +1140,10 @@ Terra.describeViewports('Select', ['tiny'], () => {
 
     describe('multiple placeholder should be shown initially', () => {
       Terra.it.validatesElement();
+
+      after(() => {
+        browser.refresh();
```

Comment:
These were removed to fix test errors when running in firefox. Does reverting this change break the firefox test runs? https://github.com/cerner/terra-core/commit/837514e05df9f889ed4dcc15b0ef68a06f0e175f#diff-80bc530bcf881e722aea305d312913ddL1143

Reply:
yeah code is reverted with browser.refresh(). Tested this in firefox and the test is passing.

Reply:
i ran our internal nighly jenkins job against this branch on firefox and it passed.  Looks good.